packages/react-microlink/src/components/Card/CardMedia/index.js

```diff
@@ -1,19 +1,37 @@
-import { createElement } from 'react'
+import React, {Component, Fragment} from 'react'
 
 import {getUrlPath} from '../../../utils'
 
 import Image from './image'
 import Video from './video'
+import {ImageLoadCatcher} from './loader'
 
 const isVideo = ({video}) => getUrlPath(video) !== null
 
-export default props => {
-  if (!isVideo(props)) {
-    return createElement(Image, {
-      className: 'microlink_card__media_image',
-      ...props
-    })
+export default class CardMedia extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loadingImage: this.props.image ? true : false
+    }
   }
 
-  return createElement(Video, props)
+  render() {
+    const {loadingImage} = this.state
+    return (
+      <Fragment>
+        {!isVideo(this.props) ? (
```

Comment:
hmm we can use here a more generic code like using `createElement`

Reply:
👌 this was sorted in #84